tests/rptest/perf/ts_read_openmessaging_test.py

```diff
@@ -23,6 +23,8 @@ def __init__(self, ctx):
             "retention_local_strict": True,
             "retention_local_target_bytes_default": 16 * 1_000_000,  # 16 MB
             "cloud_storage_spillover_manifest_size": None,
+            # approximate limit for m7gd.xlarge
+            "cloud_storage_max_throughput_per_shard": 25 * 1_000_000,
```

Comment:
Shouldn't there be some effort to scale this for the actual machine size? Does this test have the ability to run in docker (i.e., does it scale itself down in that case)?

Reply:
> Shouldn't there be some effort to scale this for the actual machine size?

None of the CDT OMB tests currently scale any of their parameters automatically with machine size. 

> Does this test have the ability to run in docker

No, it requires dedicated nodes.